feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/FilterLanguageChip.kt

```diff
@@ -0,0 +1,93 @@
+package io.github.droidkaigi.confsched2023.sessions.component
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowDropDown
+import androidx.compose.material.icons.filled.Check
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.FilterChip
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberUpdatedState
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import io.github.droidkaigi.confsched2023.model.Lang
+import io.github.droidkaigi.confsched2023.sessions.SessionsStrings.SupportedLanguages
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun FilterLanguageChip(
+    selectedLanguages: List<Lang>,
+    languages: List<Lang>,
+    onLanguagesSelected: (Lang, Boolean) -> Unit,
+    modifier: Modifier = Modifier,
+    isSelected: Boolean = false,
+    selectedLanguagesValues: String = "",
+    onFilterLanguageChipClicked: () -> Unit,
+) {
+    var expanded by remember { mutableStateOf(false) }
+    val onLanguagesSelectedUpdated by rememberUpdatedState(newValue = onLanguagesSelected)
```

Comment:
I'm not sure if we should use rememberUpdatedState here. Is there any reason why you used rememberUpdatedState while other click events don't use this 👀

Reply:
Just because FilterDayChip was implemented like that. I think just invoke the passed lambda function `onLanguageSelected` is okay... 🤔